Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file should be ignored

Reply:
Sure. Added to `.gitignore` along with the Cartfile resolved. My understanding was that these files *should* be committed however to ensure that those using the project are using the last known working versions of any dependencies.

Reply:
Sure, but Operadics is essentially frozen, and the other two are test-only dependencies.  It doesn’t matter which version our own internal repos are built against, it won’t affect our clients.